includes/admin.form.inc

```diff
@@ -92,5 +92,49 @@ function islandora_entities_settings($form, &$form_state) {
     '#required' => TRUE,
   );
 
+  $form['islandora_entities_scholar_title_solr_field'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Collection to ingest title entities in.'),
+    '#default_value' => variable_get(
+      'islandora_entities_scholar_title_solr_field',
+      'islandora:entity_collection'
+    ),
+    '#description' => t('Used by dependent modules for autocompletion.'),
+    '#required' => TRUE,
+  );
+
+  $form['islandora_entities_department_solr_field'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Collection to ingest department entities in.'),
+    '#default_value' => variable_get(
+      'islandora_entities_department_solr_field',
+      'islandora:entity_collection'
+    ),
+    '#description' => t('Used by dependent modules for autocompletion.'),
+    '#required' => TRUE,
+  );
+
+  $form['islandora_entities_disambiguated_solr_field'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Collection to ingest disambiguated full name entities in.'),
+    '#default_value' => variable_get(
+      'islandora_entities_disambiguated_solr_field',
+      'islandora:entity_collection'
+    ),
+    '#description' => t('Used by dependent modules for autocompletion.'),
```

Comment:
As per https://github.com/Islandora/islandora_solution_pack_entities/pull/74#discussion_r28295158 not sure the dependent module business is completely accurate; they are used in this module as well?